### DISCUSSION TITLE
Problems in Tourism Website

### DISCUSSION THREAD

**Discussion by AmitBarman99** [PritamSarbajna/tourism-website#52] [General]
Hey @PritamSarbajna  ,
I found some problems in this website
1> The home background image is very big size, plz compressed it.
2> In Explore section the glass effects box UI is leftside aligned. (when screen >850px)
3> The things in this explore box are also not responsive.
4> About us section should have a leftside paddings at screen >770px.
5> Contribution box are too big.
6> There is no contribution part in navbar.
7> Navbar has a few less padding.

However I am interested to solve all these.
Will you assign me to do it ?

**Comment by PritamSarbajna** [comment]
Hey @AmitBarman99 welcome again.

Thanks for pointing out these issues. To get you assigned, you have to raise issues first then I can assign you. Please try to make separate issues for all above. However You can also create a single issue and list these.

**Reply by AmitBarman99** [reply]
ok 
I will create all the issues separately.

**Reply by PritamSarbajna** [reply]
Yeah go for it 🙌

**Comment by WasiunAlam** [comment]
Hey @PritamSarbajna. I have found some bugs and areas to improve.
For Example:

1. The navbar which comes after clicking Hamburger in potrait device is not looking much good. 
2. White background in the light mode is really hurting client eyes. It need to be fixed.
3. In the website, the image don't load in a moment. It takes a lot of time.
4. Also I have found a lot of problem in the responsive design like text is going out of its parents etc.

I really want to improve these bugs and enhance the website.
With your grant permission, I can start working on these issues and solve these bugs.

Thank you for hearing me out.
